acceptance/bundle/override/test.toml

```diff
@@ -0,0 +1 @@
+Cloud = false
```

Comment:
Does it need to be in this PR?

Reply:
Yes, because introduction of NODE_TYPE_ID breaks those tests otherwise, since they use aws-specific node type.